lib/membrane_mp4/demuxer/isom/samples_info.ex

```diff
@@ -188,13 +215,16 @@ defmodule Membrane.MP4.Demuxer.ISOM.SamplesInfo do
   end
 
   defp get_chunk_samples(chunk, track_data) do
-    %{chunk_no: chunk_no, track_id: track_id} = chunk
+    %{chunk_no: chunk_no, track_id: track_id, chunk_offset: chunk_offset} = chunk
 
     {track_data, samples_no} = get_samples_no(chunk_no, track_data)
 
     Enum.map_reduce(1..samples_no, track_data, fn _no, track_data ->
       {sample, track_data} = get_sample_description(track_data)
-      sample = Map.put(sample, :track_id, track_id)
+
+      sample =
+        Map.put(sample, :track_id, track_id) |> Map.put(:chunk_offset, chunk_offset)
```

Comment:
We could accumulate the offset of each chunk here

Reply:
done